docs/docsite/_themes/sphinx_rtd_theme/ansible_banner.html

```diff
@@ -21,4 +22,27 @@
       element.appendChild(para);
       document.write('</div>');
     }
+    {% if (not READTHEDOCS) and (available_versions is defined) %}
+      // Create a banner if we're not the latest version
+      current_url_path = window.location.pathname;
+      if (startsWith(current_url_path, "/ansible/latest/") || startsWith(current_url_path, "/ansible/{{ latest_version }}/")) {
+       // no banner for latest release
+      } else if (startsWith(current_url_path, "/ansible/devel/")) {
+        document.write('<div id="banner_id" class="admonition caution">');
+        para = document.createElement('p');
+        banner_text=document.createTextNode("You are reading the *devel* version of the Ansible documentation - this version is not guaranteed stable. Use the version selection to the left if you want the latest stable released version.");
+        para.appendChild(banner_text);
+        element = document.getElementById('banner_id');
+        element.appendChild(para);
+        document.write('</div>');
+      } else {
+        document.write('<div id="banner_id" class="admonition caution">');
+        para = document.createElement('p');
+        banner_text=document.createTextNode("You are reading an older version of the Ansible documentation. Use the version selection to the left if you want the latest stable released version.");
```

Comment:
This may be having a slightly weird effect on the 2.10 pre-release documentation.  For example: https://docs.ansible.com/ansible/2.10/collections/community/aws/aws_secret_module.html

Reply:
This is a backport PR for 2.8 FWIW.

Reply:
@tremble the behavior you're seeing is an issue with the logic for the "older release" banner. Related but not addressed here . . . at least not yet.